nixos/doc/manual/installation/installing.chapter.md

```diff
@@ -1,30 +1,147 @@
 # Installing NixOS {#sec-installation}
 
-## Booting the system {#sec-installation-booting}
+## Booting from the install medium {#sec-installation-booting}
+
+To begin the installation, you have to boot your computer from the install drive.
+
+1.   Plug in the install drive. Then turn on or restart your computer.
+
+2.   Press <kbd>F12</kbd> (or the appropriate key) to open the boot menu.
```

Comment:
```suggestion
2.   Open the boot menu by pressing the appropriate key that's shown on your monitor on early boot. If no key is displayed consult your motherboard or system handbook.
```
That key differs widely across many motherboard and laptop manufactures. It could also be F1, F9, F10, Enter or Del.

Probably needs to be deduped with the warning.

Reply:
i had there first to look it up in the computers documentation, but i think it's easier to look it up online. most people probably don't find these documentations

i use your suggestion for the first sentence